contrib/dhcp_agent/dhcp_agent.py

```diff
@@ -16,13 +16,13 @@
 
 IS_PY3 = sys.version_info[0] == 3
 if IS_PY3:
-    from dbm import gnu as gdbm
+    from dbm import gnu as cache_db
```

Comment:
\[Pylint\] INFO: Import "from dbm import gnu as cache\_db" should be placed at the top of the module \[wrong\-import\-position\]